internal/server/resolvers/node.resolvers.go

```
@@ -0,0 +1,192 @@
+		return nil, err
+	}
+	return *worktreeNodeValue(w), nil
+}
+
+func resolveProcessNode(ctx context.Context, r *Resolver, id string) (graphql1.Node, error) {
+	if r.PS == nil {
+		return nil, fmt.Errorf("ps provider not configured")
+	}
+	pid, err := psprovider.ParseProcessID(id)
+	if err != nil {
+		return nil, nil
```

golangci-lint (2.12.1)

[error] error is not nil (line 112) but it returns nil

(nilerr)